src/content/quickstart/openshiftgn/_index.md

```diff
@@ -69,13 +69,12 @@ subctl deploy-broker  --kubeconfig cluster-a/auth/kubeconfig --service-discovery
 #### Join cluster-a and cluster-b to the broker
 
 ```bash
-subctl join --kubeconfig cluster-a/auth/kubeconfig broker-info.subm --clusterid west
+subctl join --kubeconfig cluster-a/auth/kubeconfig broker-info.subm --clusterid west --globalnet-cidr 169.254.0.0/19
```

Comment:
This should not be part of a quickstart, it's a feature that we have, but for very specific use cases. Specially not a "quick" start

Reply:
It was added as a precaution (and part of the comment https://github.com/submariner-io/submariner-website/pull/93#discussion_r420724783) to avoid facing race conditions that @manosnoam was facing last week.

Reply:
@mangelajo As you know, we have a known issue where same globalCIDRs can get allocated to the clusters. This will be addressed as part of https://github.com/submariner-io/submariner-operator/issues/299

Until this is fixed, lets have the --globalnet-cidr included as part of Quickstart. We can remove it once the above use-case is addressed.